Implemetar Exportação Excel de  Dados de Arquivos Enviados

AlineMaire opened the pull request.

AlineMaire committed:
Implemetar Exportação Excel de  Dados de Arquivos Enviados

ricardfo commented:
No index:
- a autorização está bloqueando o acesso ao sistema;
- quando faz a busca por nome do arquivo está dando erro;
- dá para eliminar o "if  $request->type" e fazer que o método excel já retorne a planilha.

AlineMaire committed:
Implementação de Download Planilha Excel

- Implementada a função de download planilha excel;
- Correção de erro em busca de arquivo.

Fix #14
